### PR TITLE
simplify fibonacci source code comment

### PR DESCRIPTION
**Pull request by wilzbach**

### CODE REVIEW

File: public/content/en/basics.md [hunk 1]
```diff
@@ -935,14 +935,12 @@ void main() {
...
     FibonacciRange fib;
 
-    // The magic function take creates another
-    // range which just will return N elements
-    // at maximum. This range is _lazy_ and just
-    // touches the original range if actually
-    // needed (iteration)!
+    // Take creates another range which will return
+    // N elements at maximum.
+    // This range is _lazy_ and just touches the original
```
Comment (stonemaster) [inline public/content/en/basics.md]: 48 chars

File: public/content/en/basics.md [hunk 2]
```diff
@@ -935,14 +935,12 @@ void main() {
...
 
-    // Let's test this FibonacciRange!
     FibonacciRange fib;
 
-    // The magic function take creates another
-    // range which just will return N elements
-    // at maximum. This range is _lazy_ and just
-    // touches the original range if actually
-    // needed (iteration)!
+    // Take creates another range which will return
```
Comment (stonemaster) [inline public/content/en/basics.md]: 48 chars

Comment (stonemaster) [inline public/content/en/basics.md]: `take` instead of Take (or just take)

**Comment by wilzbach** [comment]
Rebased.

**Pull request merged**